aws-marketplace-install.md

```diff
@@ -24,34 +26,55 @@ Please contact us at support@kubecost.com with any questions and we'd be happy t
 }
 ```
 
-## Step 2. Create an IAM role with the appropriate trust relationships
-We recommend doing this via eksctl. More detail and how to set up the appropriate trust relationships is available [here](https://docs.aws.amazon.com/eks/latest/userguide/create-service-account-iam-policy-and-role.html).
+## Step 2. Create an IAM role for service account (IRSA).
+We recommend doing this via [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html). The command below helps to automate these manual steps:
+- Create an IAM role with AWS managed IAM policy.
+- Create a K8s service account name `awsstore-serviceaccount` in your Amazon EKS cluster.
+- Set up trust relationship between the created IAM role with awsstore-serviceaccount.
+- Modify `awsstore-serviceaccount` annotation to associate it with the created IAM role 
+
+Please remember to replace `CLUSTER_NAME` with your actual Amazon EKS cluster name.
+
 ```
 eksctl create iamserviceaccount \
     --name awsstore-serviceaccount \
     --namespace kubecost \
-    --cluster cluster_name \
-    --attach-policy-arn IAM_policy_ARN \
+    --cluster CLUSTER_NAME \
+    --attach-policy-arn arn:aws:iam::aws:policy/AWSMarketplaceMeteringRegisterUsage \
     --approve \
     --override-existing-serviceaccounts
 ```
+For more detail and how to set up the appropriate trust relationships is available [here](https://docs.aws.amazon.com/eks/latest/userguide/create-service-account-iam-policy-and-role.html).
```

Comment:
Drop "For", can just be "more detail on how to set up the appropriate trust relationships is available here"

Reply:
I will fix it in the next commit.